libp2p/multiaddress.nim

```diff
@@ -631,6 +632,23 @@ proc validate*(ma: MultiAddress): bool =
       discard
   result = true
 
+proc init*(mtype: typedesc[MultiAddress], address: TransportAddress): MultiAddress =
```

Comment:
Please avoid usage of `nativesockets`. I have added such procedure for you in https://github.com/status-im/nim-libp2p/commit/fb9d2d7d05f5e595fa44d3bf71e2629702399471